src/main/java/org/gridsuite/modification/server/modifications/ModificationUtils.java

```diff
@@ -771,6 +785,27 @@ private String getConnectionFieldName(int feederNumber, String baseFieldName) {
         };
     }
 
+    private AttributeModification<String> getVoltageLevelId(Object modificationInfos, int feederNumber) {
+        return getConnectionDetail(modificationInfos, feederNumber,
+                BranchModificationInfos::getVoltageLevelId1, BranchModificationInfos::getVoltageLevelId2,
+                InjectionModificationInfos::getVoltageLevelId);
+    }
+
+    private AttributeModification<String> getBusOrBusbarSectionId(Object modificationInfos, int feederNumber) {
+        return getConnectionDetail(modificationInfos, feederNumber,
+                BranchModificationInfos::getBusOrBusbarSectionId1, BranchModificationInfos::getBusOrBusbarSectionId2,
+                InjectionModificationInfos::getBusOrBusbarSectionId);
+    }
+
+    private AttributeModification<String> getEquipmentId(Object modificationInfos) {
```

Comment:
It's a pity we grapple with Object type all the way. modificationInfos should be BasicEquipmentModificationInfos in processConnectivityPosition() signature